sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/ResolveGroupingAnalyticsSuite.scala

```diff
@@ -287,9 +287,9 @@ class ResolveGroupingAnalyticsSuite extends AnalysisTest {
         Seq(GroupingSets(Seq(Seq(), Seq(unresolved_a), Seq(unresolved_a, unresolved_b)))),
         Seq(unresolved_a, unresolved_b), r1))
     val expected = Project(Seq(a, b), Sort(
-      Seq(SortOrder('aggOrder.byte.withNullability(false), Ascending)), true,
+      Seq(SortOrder(grouping_a, Ascending)), true,
       Aggregate(Seq(a, b, gid),
-        Seq(a, b, grouping_a.as("aggOrder")),
+        Seq(a, b, gid),
```

Comment:
This is a more correct plan. If you look at other tests in this file, `test("grouping function")` and `test("filter with grouping function")`, they both expect plans to group by `gid` first, then calculate the grouping function (which is `grouping_a`) in agg list or the filter node above.